### DISCUSSION TITLE
Safe HTML in help attribute

### DISCUSSION THREAD

**Discussion by MichaelSchmidle** [formkit/formkit#1322] [Ideas]
Imagine a sign-up form that requires users to accept Terms of Service and Privacy Policy. Usually, these are two links to the respective pages describing the TOS and policy in detail, right there with the checkbox.

I'd like to be able to do this with FormKit and pass safe HTML (i.e., Markdown-processed) strings as help attributes to forms and form elements that actually gets rendered as HTML (probably via `v-html` directive).

And I can imagine the same being helpful for validation messages and even labels while we're at it.

Could this be implemented at some point?

**Comment by dominikklein** [comment]
You can do that with the usage of the section Schema.

https://formkit.com/essentials/inputs#sections

**Comment by fenilli** [comment]
There is no safe HTML injection that we would add to the core, but you can still do it vie `sectionSchema` previously mentioned, there is an example in the examples page: https://formkit.link/de8e6530b80717f5229813258e739e7b

**Comment by MichaelSchmidle** [comment]
Thanks for your kind responses and links! 🙏
